src/LogNormal/LogNormal.sol

```diff
@@ -39,6 +39,17 @@ struct LogNormalParams {
     address controller;
 }
 
+/// @dev Thrown when the mean parameter is not within the allowed bounds.
+error InvalidMean();
+
+/// @dev Thrown when the width parameter is not within the allowed bounds.
+error InvalidWidth();
+
+uint256 constant MIN_WIDTH = 1;
```

Comment:
I think this is fine, nothing will work in the boundary scenarios anyway so I'm unsure if we should even worry about writing tests to discover "real" bounds.